test/e2e/customconfigs/skipjobswithoutqueuename_test.go

```diff
@@ -237,7 +239,11 @@ var _ = ginkgo.Describe("ManageJobsWithoutQueueName", ginkgo.Ordered, func() {
 			var testPod, createdPod *corev1.Pod
 			var podLookupKey types.NamespacedName
 			ginkgo.By("creating a pod without a queue name", func() {
-				testPod = testingpod.MakePod("test-pod", ns.Name).Obj()
+				testPod = testingpod.MakePod("test-pod", ns.Name).
+					Image(util.E2eTestAgnHostImage, util.BehaviorWaitForDeletion).
+					Request(corev1.ResourceCPU, "1").
+					Request(corev1.ResourceMemory, "2Gi").
+					Obj()
```

Comment:
Is this necessary for the test to work?

Reply:
I added this for consistency with the following tests and because I understood that the decision of gating the pod could be influenced by the resources it requests.  

Now that I re-read the code, I see that, in this case, the evaluation of `manageJobsWithoutQueueName` is taken at webhook time and the pod spec won't be considered at all. In such case I think that piece of code can be dropped.

Did I understand correctly?